ballsdex/settings.py

```diff
@@ -435,6 +444,11 @@ def update_settings(path: "Path"):
   - ballsdex.packages.trade
 """
 
+    if add_spawn_chance_range::
```

Comment:
That doesn't compile
```suggestion
    if add_spawn_chance_range:
```

Reply:
Oh well this was fixed already